etc/geoserver/workspaces/MobiData-BW/ipl-db/sharing_stations_bicycle/featuretype.xml

```diff
@@ -100,6 +100,15 @@ WHERE st.num_bicycles_available &gt;= 0
         <de>Name des Fahrzeugs</de>
       </description>
     </attribute>
+    <attribute>
+      <name>capacity</name>
+      <minOccurs>0</minOccurs>
+      <maxOccurs>0</maxOccurs>
```

Comment:
maxOccurs of 0 seems incorrect

Reply:
> maxOccurs of 0 seems incorrect

Currently the highest capacity is 50 but this obviously can change again. What do you suggest? Is `minOccurs `and `maxOccurs `mandatory?

Reply:
Done.